### PR TITLE
Implement face rect detection

### PR DESCRIPTION
**Pull request by svetlanama**
@KimDarren Please review implementation of “face rect detection”

### CODE REVIEW

**Comment by KimDarren** [comment]
Thanks for your work. I'll review your code asap..!
Give me a second 😄

File: FaceCropper/Classes/FaceCropper.swift [hunk 1]
```diff
@@ -85,6 +85,41 @@ public extension FaceCropper where T: CGImage {
...
+      }
+      
+      completion(.success(result))
+    })
+    
+    let detectFaceRequestHandler = VNImageRequestHandler(cgImage: self.detectable, options: [:])
+    do {
+      try detectFaceRequestHandler.perform([detectFaceRequest])
+    } catch {
+      print(error)
```
Comment (KimDarren) [inline FaceCropper/Classes/FaceCropper.swift]: I think to print error is not needed in this library, and the completion handler have to be called when any error occurred like this,
```swift
completion(.failure(error))
```

Comment (KimDarren) [reply FaceCropper/Classes/FaceCropper.swift]: To print error is not what this library need to do. It's just what the developers using this library need to do if they want. 😄

**Review by KimDarren — CHANGES_REQUESTED** [review]

File: FaceCropper/Classes/FaceCropper.swift (hunk 1 above)
Comment (svetlanama) [reply FaceCropper/Classes/FaceCropper.swift]: @KimDarren  Done!

**Pull request closed without merging**